webapp/src/polyfills.ts

```diff
@@ -1,7 +1,7 @@
 /***************************************************************************************************
  * Load `$localize` onto the global scope - used if i18n tags appear in Angular templates.
  */
-import '@angular/localize/init';
+// import '@angular/localize/init';
```

Comment:
@rahul-rathore-576 if it's not relevant in new Angular version, let's just remove, no need to leave it commented out

Reply:
@evereq I have removed `@angular/localize` from polyfills.ts.